test/test_utils/__init__.py

```diff
@@ -1211,6 +1211,29 @@ def get_cuda_version_from_tag(image_uri):
     return cuda_framework_version
 
 
+def get_cuda_major_minor_version_from_tag(image_uri):
+    """
+    Get cuda version from tag (i.e. cu111) and convert it to major.minor (11.1)
+    @param image_uri: ECR iamge URI
+    @return: major.minor cuda version
+    """
+    cuda_framework_version = None
```

Comment:
nit: some code duplication here. let me use the other utility function

Reply:
this helper function is no longer used in this PR now.. do you plan to resolve duplication or remove this helper function altogether?